releasenotes/notes/0.13/defaults_TranspileConfig-4109a90c278d46df.yaml

```diff
@@ -0,0 +1,30 @@
+---
+upgrade:
+  - |
+    The ``TranspileConfig`` class which was previously used to set
+    run time configuration for a :class:`qiskit.transpiler.PassManager` has
+    been removed and replaced by a new class
+    :class:`qiskit.transpile.PassManagerConfig`. This new class has been
+    structured to include only the information needed to construct a
+    :class:`~qiskit.transpiler.PassManager`. The attributes of this class are:
+
+    * ``initial_layout``
+    * ``basis_gates``
+    * ``coupling_map``
+    * ``backend_properties``
+    * ``seed_transpiler``
+
+deprecations:
+  - |
+    The ``callback`` kwarg of the :class:`qiskit.transpiler.PassManager` class's
+    constructor has been deprecated and will be removed in a future release.
+    Instead of setting it at the object level during creation it should now
+    be set as a kwarg parameter on the :meth:`qiskit.transpiler.PassManager.run`
+    method.
+  - |
+    The function ``transpile_circuit`` in
+    :mod:`qiskit.transpiler` is deprecated and will be removed in a future
```

Comment:
Looks like this was removed, not deprecated.

Reply:
Good catch, I'll fix this now

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4057/commits/31da6c99ba275c899eaf6f8483d5edac2b596efb